### ISSUE TITLE
[Feature request] Possibility to type the date

### ISSUE THREAD

**Issue by vicenterusso** [mengxiong10/vue2-datepicker#594]
**What problem does this feature solve?**
Gives the user the possibility to type the date without having to use the mouse, and of course with date-range support

If it's not trivial, please give some guidance and I'll fork and try myself

**Comment by mengxiong10** [comment]
Have it already

**Comment by vicenterusso** [comment]
That's correct, my mistake. Also, I tried to mask the typing values (to avoid user typing "-" or "/" or even the separator.. could this be done without too much work?
